ICS4: Create 32 byte commitment (#1168)
cosmos/ibc: spec/core/v2/ics-004-packet-semantics/PACKET.md

All acknowledgements must be committed to and stored under the ICS24 acknowledgment path.

```typescript
// commitV2Acknowledgement hashes each app acknowledgment and hashes them together

> // the final preimage is then prepended with the byte 0x02 in order to clearly define the protocol version

Reviewer: ```suggestion
// the final preimage will be prepended with the byte 0x02 before hashing in order to clearly define the protocol version
```
